.github/workflows/ci-unix-shared-installed.yml

```diff
@@ -35,11 +35,19 @@ jobs:
       working-directory: ./ext
       run: sed -i'' -e 's/-DBUILD_SHARED_LIBS=OFF/-DBUILD_SHARED_LIBS=ON/' *.cmd
     - name: Add recent repository for libyuv-dev
+      if: runner.os == 'Linux'
       run: echo "deb http://azure.archive.ubuntu.com/ubuntu/ jammy main restricted universe multiverse" | sudo tee -a /etc/apt/sources.list
     - name: Update apt
+      if: runner.os == 'Linux'
       run: sudo apt-get update
-    - name: Install dependencies
+    - name: Install dependencies (Linux)
+      if: runner.os == 'Linux'
```

Comment:
Optional: We can combine lines 37-45 into the "Install dependencies (Linux)" step.